src/Annotation/ServiceDefinition.php

```diff
@@ -79,4 +79,11 @@ class ServiceDefinition extends Plugin {
    */
   public $contexts;
 
+  /**
+   * The successful response code for this definition.
+   *
+   * @var int
+   */
+  public $response_code = 200;
```

Comment:
200 being a default value only? Or, is this hardcoded and a TODO to make this return an appropriate error code?